en/micro-integrator/docs/references/connectors/ldap-connector/ldap-connector-example.md

```diff
@@ -0,0 +1,214 @@
+# LDAP Connector Example
+
+The LDAP connector allows you to connect to any LDAP server through a simple web services interface and perform CRUD 
+(Create, Read, Update, Delete) operations on LDAP entries. This connector uses the JAVA JNDI APIs to connect to a 
+required LDAP server.
+
+Given below is a sample scenario that demonstrates how to perform CRUD operations on LDAP entries using LDAP Connector.
+
+## What you'll build
+
+This example demonstrates on how to use the ldap connector to create and read ldap entries on a student directory. 
+
+![image](../../../assets/img/connectors/ldap_connector/ldap_connector_usecase.png)
+
+This will have 2 API resources, `create`, `search`.
+
+todo : add an image
+
+* `/create` : This will create a new LDAP entry in the LDAP server.
+
+* `/search` : This will performs a search for one or more LDAP entities with the specified search keys.
+
+## Configure the connector in WSO2 Integration Studio
+
+Follow these steps to set up the ESB Solution Project and the Connector Exporter Project. 
+
+{!references/connectors/importing-connector-to-integration-studio.md!} 
+
+1. Right click on the created ESB Solution Project and select, -> **New** -> **Rest API** to create the REST API. 
+   
+2. Provide the API name as `college_student_api` and the API context as `/student`. You can go to the source view of the 
+xml configuration file of the API and copy the following configuration. 
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<api context="/student" name="college_student_api" xmlns="http://ws.apache.org/ns/synapse">
+    <resource methods="POST" url-mapping="/create">
+        <inSequence>
+            <sequence key="init_sequence"/>
+            <sequence key="add_student_sequence"/>
+        </inSequence>
+        <outSequence/>
+        <faultSequence/>
+    </resource>
+    <resource methods="POST" url-mapping="/search">
+        <inSequence>
+            <sequence key="init_sequence"/>
+            <sequence key="search_student_sequence"/>
+        </inSequence>
+        <outSequence/>
+        <faultSequence/>
+    </resource>
+</api>
+```
+   
+3. Right click on the created ESB Solution Project and select, -> **New** -> **Sequence** to create the following 
+sequences.
+
+* init_sequence - `<ldap.init>` element authenticates with the LDAP server in order to gain access to perform various 
```

Comment:
These need to be tabbed under the list number or it does not display as part of the list.

Reply:
Updated